CMakeLists.txt

```diff
@@ -79,7 +79,12 @@ if(NOT HAS_FILESYSTEM)
   check_include_file_cxx("experimental/filesystem" HAS_EXPERIMENTAL_FILESYSTEM)
 endif()
 
-if(NOT HAS_FILESYSTEM AND NOT HAS_EXPERIMENTAL_FILESYSTEM)
+if(HAS_FILESYSTEM) 
+  list(APPEND CHIP_SPV_DEFINITIONS "USE_FILESYSTEM")
+elseif(HAS_EXPERIMENTAL_FILESYSTEM)
+  list(APPEND CHIP_SPV_DEFINITIONS "USE_EXPERIMENTAL_FILESYSTEM")
+  link_libraries(stdc++fs)
+elseif(NOT HAS_FILESYSTEM AND NOT HAS_EXPERIMENTAL_FILESYSTEM)
```

Comment:
This was already implemented via Config.hh.in (Config.hh in the build directory).